### PR TITLE
Trash and Services

### PR DESCRIPTION
**Pull request by Rohitprasad83**
## Trash Page for deleted notes
-  added trash page with deleted notes
- added Tags to note
- moved API calls to services folder

J7nZE4CxPW

### CODE REVIEW

**Review by SanketDhabarde — APPROVED** [review]
- liked how you separated services for every function
- Also Separating the utils
- Good folder structure

File: src/context/note-context.jsx [hunk 1]
```diff
@@ -13,11 +13,12 @@ const NoteContextProvider = ({ children }) => {
...
     pinned: false,
+    tags: '',
   })
 
   const [notes, setNotes] = useState([])
   const [archiveNotes, setArchiveNotes] = useState([])
-
+  const [trash, setTrash] = useState([])
   return (
     <noteContext.Provider
```
Comment (surbhi84) [inline src/context/note-context.jsx:23]: you can use camelcasing ( i.e first letter capital as well) for naming contexts as we usually use smallcase for html elements in react.

**Review by surbhi84 — APPROVED** [review]
The code is clean which makes it readable.
Good names are used that makes the logic come out clearer.

File: src/context/note-context.jsx (hunk 1 above)
Comment (Rohitprasad83) [reply src/context/note-context.jsx:23]: will do it

**Pull request merged**